### ISSUE TITLE
[FEAT]: please add options for sorting video's by folders 

### ISSUE THREAD

**Issue by allmightyasian** [nova-video-player/aos-AVP#893]
### Description

Like mx player please add a options for sorting video's by folders in homepage

**Comment by allmightyasian** [comment]
Also please add myanimelist support,

**Comment by sdugoten** [comment]
Yes, need sorting on a peer folder basis....right now, the sorting order is global.....

Currently you can change the sorting inside a folder, but that setting won't be remembered.

**Comment by Raitsa** [comment]
Yes, on my nas I sort stuff by date, newest  content is up. But when I go to a folder, I would like to sort by name, lets say ten episodes of some series, would love to see them sorted automatically from  1 to 10. 
This kind of sorting works in kodi.
